core/pom.xml

```diff
@@ -229,5 +229,23 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>google-conscript</id>
```

Comment:
What is this profile for?

Reply:
This profile allows you to run the Postgres JDBC integration tests using Conscrypt crypto.